AnnoyancesFilter/sections/push-notifications.txt

```diff
@@ -470,6 +470,7 @@ ficbook.net#$##notificationsSubscribeModal { display: none!important; }
 ficbook.net#$#body { overflow: visible!important; padding-right: 0!important; }
 ficbook.net#$##notificationsSubscribeModal ~ .modal-backdrop { display: none!important; }
 indiatimes.com##.notification_popup
+||theswagsports.com/aud/message
```

Comment:
Not working on my end. Just not shown after page refresh, until service-workers are stopped.
`/firebase-messaging.js$domain=` will help.

<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/165386347-0d2a41c7-6c0b-454c-97fc-33bfccabed5d.png)

</details>